src/cache/mesi_controller.py

```diff
@@ -68,8 +72,8 @@ def handle_snoop(
                 MESIState.MODIFIED,
             ]:
                 warnings.warn(
-                    f"Not possible WRITE-BACK operation on address \
-                    {address:08x} in {current_state.name} state of our LLC",
+                    f"Not possible WRITE-BACK operation on address"
+                    f" {address:08x} in {current_state.name} state of our LLC",
```

Comment:
Should this one also have the warning color, or is there a convention I'm missing?

Reply:
I fixed this.